Kitodo/src/main/java/org/kitodo/production/services/data/BaseBeanService.java

```diff
@@ -13,24 +13,24 @@
 
 import io.reactivex.annotations.CheckReturnValue;
 
-import java.util.AbstractCollection;
 import java.util.EnumMap;
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
-import java.util.regex.Pattern;
 
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
-import org.hibernate.query.Query;
 import org.kitodo.data.database.beans.BaseBean;
 import org.kitodo.data.database.exceptions.DAOException;
 import org.kitodo.data.database.persistence.BaseDAO;
 import org.primefaces.model.SortOrder;
```

Comment:
Is `org.primefaces.model.SortOrder` correct in this place? Its look for me like a wrong selected class which provides the same content as the original needed class. But maybe I'm wrong here.

Reply:
Since the sorting is selected from the user interface, the sorting enum is still used in the source code. It is then converted to string `ASC` or `DESC`.

Reply:
Understandable and should be changed in future on more separating frontend/UI from backend.